src/OpenTK.GLWidget/OpenTK.GLWidget.csproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
```

Comment:
This specific change is an alteration intended to work around a bug in MSBuild 15 - under some circumstances, it may fail to recognize GTK libraries correctly. Version 14 does not have this problem. Even so, this is merely a hint to the compiler.